themes/wim/components/wim/_block--menu-top-tasks.scss

```diff
@@ -12,7 +12,7 @@
       min-height: 170px;
     }
     .menu {
-      background-color: #efefef;
+      background-color: #efefef; //@todo use variable
```

Comment:
@Crazylord We discussed this a few week ago. Make use of proper variables so we can make changes to future themes more easily. Go through all the files and replace the static hex values for the correct variables.